src/modules/75-cf/pages/feature-flags/components/NoFeatureFlags.tsx

```diff
@@ -89,17 +89,26 @@ export const NoFeatureFlags: React.FC<NoFeatureFlagsProps> = ({
     }
   }
 
+  if (hasSearchTerm || hasFlagFilter) {
+    return (
+      <Container flex={{ justifyContent: 'center' }} padding="xxxlarge">
+        <NoData
+          imageURL={hasFeatureFlags && (hasSearchTerm || hasFlagFilter) ? noResultsImg : noFlagsImg}
+          message={mainMessage()}
+          description={additionalContent()}
+          buttonText={buttonText()}
+          buttonProps={buttonProps()}
+          padding="xxxlarge"
+          width="570px"
+        />
+      </Container>
+    )
+  }
```

Comment:
Nitpick, but try to keep whitespace around conditions
```suggestion
  }

```